recipe/bld.bat

```diff
@@ -5,6 +5,7 @@ cd build
 if errorlevel 1 exit /B 1
 
 cmake -G "NMake Makefiles" ^
+    -DCMAKE_BUILD_TYPE=STRING:RELEASE ^
```

Comment:
I wonder if `%CMAKE_ARGS%` should be added and if that maybe includes the release flag.

Reply:
Not sure, but I think explicit is better than implicit.